iroh/src/magicsock/transports/relay.rs

```diff
@@ -304,26 +307,19 @@ impl RelaySender {
     }
 }
 
-/// Split a transmit containing a GSO payload into individual packets.
-///
-/// This allocates the data.
-///
-/// If the transmit has a segment size it contains multiple GSO packets.  It will be split
-/// into multiple packets according to that segment size.  If it does not have a segment
-/// size, the contents will be sent as a single packet.
+/// Translate a UDP transmit to the `Datagrams` type for sending over the relay.
 // TODO: If quinn stayed on bytes this would probably be much cheaper, probably.  Need to
 // figure out where they allocate the Vec.
```

Comment:
Feel free to remove this comment...  I remember writing this.  At some point Quinn was using bytes in a Transmit I think.  But this is anyway a tricky thing because we cross the `IoSlice` boundary as well.  I don't think it deserves being so prominent in the code.